src/Hazelcast.Net/CP/CPSubsystem.cs

```diff
@@ -163,8 +163,9 @@ public static (string groupName, string objectName, string fullName) ParseName(s
 
             if (groupName.Length == 0)
                 throw new ArgumentException("CP group name cannot be an empty string.", nameof(name));
-
+#pragma warning disable CA1847 //Use string.Contains(char) instead
```

Comment:
There is a contradiction actually. If we use `char` then it will ask to to add comparison parameter but `string.Contains(char)` doesn't have overload for `string.Contains(char, StringComparision.XXXX)`

Reply:
my bad, what I'd do then, is use the `groupName.Contains('@', StringComparison.OrdinalIgnoreCase)` syntax here and then in `Polyfills/StringExtensions.cs` I'd add the following function in the `#if` block
```
public bool Contains(this string s, char c, StringComparison comparisonType)
    => s.Contains(new string(c), comparisonType);
```

Reply:
Added the extension at https://github.com/hazelcast/hazelcast-csharp-client/pull/682/commits/fdd2f7002d520a65a8fb4a501d422bd923d02334

Reply:
ok